todo/src/reducers/index.js

```diff
@@ -0,0 +1,40 @@
+import { GET_TODOS, ADD_TODO, COMPLETED_TODO } from "../actions/index";
+
+const initialState = {
+  todos: [
+    {
+      task: "Do this thing",
+      completed: false
+    }
+  ]
+};
+
+const todosReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case GET_TODOS:
+      return {
+        ...state
+      };
+    case ADD_TODO:
+      return {
+        ...state,
+        todos: [...state.todos, { task: action.payload, completed: false }]
```

Comment:
```suggestion
        todos: [...todos, { task: action.payload, completed: false }]
```

Reply:
IF ^^^ we changed our state structure to naming it "todos" & that value would immediately be our array